Source/Core/DolphinQt2/GameList/GameListModel.cpp

```diff
@@ -200,7 +200,7 @@ QSharedPointer<GameFile> GameListModel::GetGameFile(int index) const
   return m_games[index];
 }
 
-void GameListModel::UpdateGame(QSharedPointer<GameFile> game)
+void GameListModel::UpdateGame(const QSharedPointer<GameFile>& game)
```

Comment:
I'm not sure why it's passing around shared pointers in the first place, it probably would be fine with just a normal value, but I haven't dug that deeply into it :/

Passing a shared pointer by const ref saves a copy, though I just noticed that, inside the function, this can be moved anyways, so there's no point. I'll change this.